### PR TITLE
Add replace_html to tgwindows (for real this time)

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

direct sequel to #67103

Quite possibly, I am the stupidest motherfucker of all time. The lord himself cannot even comprehend the bounds of my stupidity and ignorance, and on judgement day, I do not know what I will say to Him. I will cower and pray. Perhaps weep.

What was the issue? What was the problem that caused tgui to fail across all servers, leaving nothing for nobody? A single parentheses.

I left a single `)` on the closing of the function. That was my fatal mistake. That was the issue. 

Fuck.

Don't merge this until I get home to test it, but this was 100% the issue.

## Why It's Good For The Game

I am stupid and forgot something simple which broke the servers.

### CODE REVIEW

**Comment by RigglePrime** [comment]
Phenomenal

**Comment by MrMelbert** [comment]
- [ ] I tested this PR

**Comment by magatsuchi** [comment]
> Here's a free checkbox
> 
> * [ ]  I tested this PR

i press it

File: code/modules/tgui/tgui_window.dm [hunk 1]
```diff
@@ -311,6 +311,18 @@
...
+ * public
+ *
+ * Replaces the inline HTML content.
+ *
+ * required inline_html string HTML to inject
+ */
+/datum/tgui_window/proc/replace_html(inline_html = "")
+	client << output(url_encode(inline_html), is_browser \
+		? "[id]:replaceHtml" \
+		: "[id].browser:update")
```
Comment (stylemistake) [inline code/modules/tgui/tgui_window.dm:324]:
```suggestion
		: "[id].browser:replaceHtml")
```

Comment (magatsuchi) [reply code/modules/tgui/tgui_window.dm]: sussy

**Pull request merged**